uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -1220,8 +1212,8 @@ public function testFindOverlappingLeaveRequestsBetweenTOILAndNonTOILRequests()
   public function testFindOverlappingLeaveRequestsForHalfDayMultipleDayRequests() {
     $contactID = 1;
 
-    $testFromDate = '2018-04-13 00:00:00';
-    $testToDate = '2018-04-15 23:59:00';
+    $testFromDate = '2018-04-13';
+    $testToDate = '2018-04-15';
```

Comment:
Why remove the time?

Reply:
`...ForHalfDayMultipleDayRequests...` - requests are in days, no time needed.